README.md

```diff
@@ -99,7 +100,11 @@ The following list represents the set of components (and props) that don't curre
 
 As new core components are released, which support referencing assets, we'll update this list to ensure users know what exactly they can expect to update using CodePush.
 
+*Note: CodePush works correctly with Video component only if using `require`. Example:*
```

Comment:
I'd reword this slightly to:
*Note: CodePush only works with Video components when using `require` in the source prop. For example:*

Reply:
Thank you!